src/PaperG/FirehoundBlob/Facebook/Targeting/FacebookDemographicTargeting.php

```diff
@@ -85,6 +85,17 @@ public function getMinAge()
         return $this->minAge;
     }
 
+    public function isValid()
+    {
+        if ($this->getMinAge() < 18 ||
```

Comment:
Similar to your other isValid methods, if you have a single simple condition, you could simply return the condition's result.